src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -2839,7 +2839,7 @@ mobile_apis::Result::eType MessageHelper::ProcessSoftButtons(
   using namespace smart_objects;
 
   if (!message_params.keyExists(strings::soft_buttons)) {
-    return mobile_apis::Result::SUCCESS;
```

Comment:
@EKuliiev Return back namespace as we have Result in different namespaces

Reply:
@AByzhynar 
1) Another Result name defined in NsSmartDeviceLinkRPC::V1 scope.
2) There are lot of such usage cases in this file. No ambiguity.

Reply:
@EKuliiev At this moment - yes. But this is possible way to errors. Consider such cases and use explicit namespaces.

Reply:
@AByzhynar, all review issues are done.
Please review.